backend/src/openarchiefbeheer/conf/ci.py

```diff
@@ -26,6 +24,7 @@
 E2E_PORT = config("E2E_PORT", default=8000)
 E2E_SERVE_FRONTEND = config("E2E_SERVE_FRONTEND", default=False)
 
+SESSION_ENGINE = "django.contrib.sessions.backends.db"
```

Comment:
But does this mean that in production it's still cache, therefore we might logout in save (in signals.py)?